Include the AppVeyor build number in the package version

qmfrederik opened the pull request:
Currently all NuGet packages which are built during the AppVeyor CI builds have the same version, `0.13-alpha`.

This PR updates the packaging process so that they include the AppVeyor build number.
The `VersionPrefix` and `VersionSuffix` are set in a `common.props` file. The `VersionSuffix` is overriden during the AppVeyor build.

This makes it easier for users (like me 😄) to consume interim versions of DiscUtils, either from the AppVeyor NuGet repository (if enabled) or an internal NuGet package feed.

Using AppVeyor build numbers works; a more structural solution may be to use a packaging like NerdBank.GitVersioning.

LordMike commented:
In some files, you took out VersionPrefix, but in others you left it and removed AssemblyTitle ... why?

Other than that. Solid idea 👍

qmfrederik commented:
> In some files, you took out VersionPrefix, but in others you left it and removed AssemblyTitle ... why?

Ah, that was a mistake. Thanks for catching it! Fixed it.

LordMike commented:
Looks good to me.
